README.md

```diff
@@ -36,12 +36,20 @@ pip3 install --user .
 ```
 
 ## Using it
-After [installing it](#installing-it), you'll need to edit your Django `settings.py` file
-and particularly the `DATABASES` section to point to an EXISTING database, of the format:
+After [installing it](#installing-it), you'll need to edit your Django `settings.py` file:
+* Add `django-spanner` as the very first app in the `INSTALLED_APPS` section
+* Edit the `DATABASES` section to point to an EXISTING database, of the format:
+
+Note: `django_spanner` MUST be added as the first entry into `INSTALLED_APPS`.
```

Comment:
This seems redundant to two lines earlier.

Reply:
Ack, thanks!